tests/modules/rust-logger/bindings.rs

```diff
@@ -0,0 +1,137 @@
+mod wasi_log {
```

Comment:
This file doesn't need to be in source control.

Reply:
removed